models/cardModel.js

```diff
@@ -233,6 +233,58 @@ function getTime(){
     return year.toString() + month.toString() + date.toString() + "_" + hours + ":" + minutes
 }
 
+// check category of the card is shared
+async function checkCardIsShared(sendValue) {
+    const query = `select count(card_id) as cnt from huco.card where card_img_path=? and card_id!=?;`;
```

Comment:
통일성을 위해 이렇게 변경하는 것은 어떤가요? 혹시 모르니 실행이 잘 되는지도 확인해주세요!
```suggestion
    const query = `SELECT count(card_id) AS cnt FROM card WHERE card_img_path=? and card_id!=?` ;
```

Reply:
좋습니다! 실행 잘 되는것도 확인했습니다